lib/puppet/pops/issues.rb

```diff
@@ -58,7 +58,8 @@ def format(hash ={})
   # in the issue's message producing block.
   # @api private
   #
-  class MessageData
+  class MessageData < Puppet::Pops::LabelProvider
```

Comment:
Yeah, so that is a quick way of getting the methods. But it is not right :-)
This class is not allowed to have those methods itself as it is language agnostic. It is the label provider's job to do that. We happen to have an english provider that has a base implementation for all different kinds of english label providers.

The support should be implemented as delegation to the label provider that is referencing.